requirements/default.txt

```diff
@@ -3,7 +3,7 @@ zarr>=2.3.0
 numcodecs!=0.6.4
 fsspec>=0.3.3
 imageio>=2.5.0
-ipykernel==5.1.1
+ipykernel>=5.1.3
```

Comment:
Do we need 5.1.3 as opposed to 5.1.1 or earlier?

Reply:
ooo you're right no `>= 5.1.1` should be fine. I'll change.